src/components/StaffList/index.jsx

```diff
@@ -33,17 +95,108 @@ class StaffList extends React.Component {
     }, () => this.props.input.onChange(this.state.staffList));
   }
 
+  /**
+   * The callback method for when a user enters a value in the autocomplete list.
+   * Responsible for updating the redux state of selected options.
+   *
+   * @param event - unused but a required parameter for autosuggest component
+   * @param suggestion - the entered suggestion
+   */
+  onSuggestionEntered(event, { suggestion }) {
+    const { staffList } = this.state;
+    const newInstructor = (suggestion.id === 'new');
+    const instructorAlreadySelected =
+      staffList.some(staff => staff.uuid === suggestion.text.uuid);
+    if (instructorAlreadySelected) {
+      // do we want to message the user?
+    }
+    const newStaffList = Array.from(this.state.staffList);
+    if (!instructorAlreadySelected && !newInstructor) {
+      newStaffList.push(suggestion.text); // add to component state
+    }
+    // add to form state (trigger change action)
+    this.setState({
+      staffList: newStaffList,
+    }, () => this.props.input.onChange(this.state.staffList));
+    // clear search string to allow for another staffer to be added.
+    this.setState({ searchString: '' });
+  }
+
+  /**
+   * The callback method for when a user selects a suggestion in the autocomplete list.
+   * Responsible for updating the search input text after selection
+   *
+   * @param selectedValue - the selected autocomplete option
+   * @returns {string} - the value for the search input box after a selection
+   */
+  getSuggestionValue(selectedValue) {
+    if (selectedValue.link_text) {
+      return selectedValue.link_text;
+    }
+    return `${selectedValue.text.given_name} ${selectedValue.text.family_name}`;
+  }
+
+
   handleRemove(uuid) {
     this.setState({
       staffList: this.state.staffList.filter(staffer => staffer.uuid !== uuid),
     }, () => this.props.input.onChange(this.state.staffList));
   }
 
+  handleSelectionEnterEvent(event) {
+    if (event.key === 'Enter') {
+      event.preventDefault();
+    }
+  }
+
+  /**
+   * Renders a suggestion in the autocomplete 'div'.  Called for each suggestion returned from
+   * the fetch response.
+   *
+   * @param suggestion - the suggestion object in the state.suggestions array
+   * @returns {*} - the rendered suggestion
+   */
+  renderSuggestion(suggestion) {
+    return (
+      <div className="d-flex flex-row m-1 p-1">
+        <div className="m-1 p-1">
+          <img
+            src={suggestion.text.profile_image_url}
+            alt={`profile for ${suggestion.text.given_name} ${suggestion.text.family_name}`}
```

Comment:
Just remembered that you can use https://github.com/edx/publisher-frontend/blob/7ed8d264bd28283ab915a46d23cfd1885383c981/src/components/Staffer/index.jsx#L6 to avoid putting `null` into the string if the family name is missing